core/dylib/src/main/java/ch/cyberduck/core/preferences/ApplicationPreferences.java

```diff
@@ -84,7 +86,13 @@ protected void setFactories() {
         this.setDefault("factory.notification.class", NotificationCenter.class.getName());
         this.setDefault("factory.iconservice.class", WorkspaceIconService.class.getName());
         this.setDefault("factory.filedescriptor.class", LaunchServicesFileDescriptor.class.getName());
-        this.setDefault("factory.schemehandler.class", LaunchServicesSchemeHandler.class.getName());
+        if(Factory.Platform.osversion.matches("10\\.(12|13|14|15|16).*")) {
```

Comment:
What about Big Sur (11.x)? It seems that it is correctly reported as of JDK 17, refer to https://bugs.openjdk.org/browse/JDK-8253702 and https://github.com/openjdk/jdk/commit/66757750

Reply:
Test the assumption in #13540.